docs/source/command.rst

```diff
@@ -62,10 +62,9 @@ protein-ligand complexes based in the 3D structure. The
               specifying mutation labels which is used to modify
 	      <filename> according to, e.g. N25R/N181D (default: None)
 			
-.. option::  -v VERSION_LABEL, --version VERSION_LABEL
+.. option::  --version
 	     
-             specifying the sub-version of propka [Jan15/Dec19]
-	     (default: Jan15)
+	      show program's version number and exit
```

Comment:
Functionality has changed; I would normally add a

```reST
.. versionchanged:: New.Version.Number
   Previous versions of PROPKA allowed selection of different sub-versions of
   PROPKA with the ``--version`` flag. This behavior was changed to simply
   display the current version of the program.
```

Reply:
Very good point, I will add this!

Reply:
Actually, I'm a bit lost where to put this :confused: You are right that there is no CHANGELOG, and adding it to the docstring of `propka/lib.py` is probably useless. I think it should show up on this page: https://propka.readthedocs.io/en/latest/command.html

Reply:
see changelog PR https://github.com/jensengroup/propka/pull/115